woocommerce/woocommerce
# Add `popoverProps` to DatePicker

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR adds popoverProps to DatePikcer.

GB changed the dropdown/popover logic so it gets rendered by default outside of the element tree at the bottom of the page. This was causing a styling not to be applied, as it wasn’t part of that tree anymore.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Before checking out this branch, go to Analytics > Settings
2. Under `Import historical data`, click the dropdown and select `Custom`.
3. Select a date under `Beginning on`.
4. The date won't be saved.
5. Checkout this branch and repeat the steps above.
6. The datepicker should work as expected.
7. Now enable the product editor using **WooCommerce > Settings > Advanced > Features**
8. Go to **Products > Add New > Pricing**
9. Add a `List price` and a `Sale price`
10. Enable `Schedule sale` and select a date for `From` and `To`.
11. It should be displayed correctly.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
